scripts/blt/deploy/.gitignore

```diff
@@ -11,6 +11,7 @@ local.blt.yml
 # Ignore build artifacts.
 /tmp/
 /reports/
+/travis_wait*
```

Comment:
Travis sometimes generates this log file during tests. It will prevent deploys unless specifically excluded from Git. Plus, no one actually wants to deploy this file.